src/frontend/components/NewPlan.tsx

```diff
@@ -219,6 +234,113 @@ const MapView: React.FC<MapViewProps> = ({ plans, isLoaded, loadError }) => {
   );
 };
 
+interface AutocompleteInputProps {
```

Comment:
Great! I was trying to combine the add destination box and the selected destinations box, aiming for: (1) hover on a destination to show details, (2) click on blank space to add a new destination (with a fuzzy match dropdown excluding already selected ones), and (3) click ‘x’ on a destination to remove it. I couldn’t get (2) to work yet, but it’s not important for now.